src/js/common/schemaform/definitions/autosuggest.js

```diff
@@ -0,0 +1,27 @@
+import _ from 'lodash/fp';
+
+import AutosuggestField from '../fields/AutosuggestField';
+
+export const schema = {
+  type: 'object',
+  properties: {
+    id: { type: 'any' },
```

Comment:
What are the different types that this can be?

Reply:
Whatever is necessary for the form/api being used. I didn't want to assume anything. For this form, it's a string.